frontend/components/AddressBook/components/AddressList/component/Address/components/DefaultCard/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@shopgate/pwa-common/components/Grid';
+import TickIcon from '@shopgate/pwa-ui-shared/icons/TickIcon';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import Portal from '@shopgate/pwa-common/components/Portal';
+import * as portals from './../../../../../../../../constants/Portals';
```

Comment:
You can import starting from the root `import ... from '@shopgate/user/constants/Portals'`